schema-serializer/src/main/java/io/confluent/kafka/formatter/SchemaMessageReader.java

```diff
@@ -164,6 +140,38 @@ protected abstract ParsedSchema parseSchema(
       List<SchemaReference> references
   );
 
+  private String getSchemaString(Properties props, boolean isKey) {
+    String propKeyRaw = isKey ? "key.schema" : "value.schema";
+    String propKeyFile = isKey ? "key.schema.file" : "value.schema.file";
+    if (props.containsKey(propKeyRaw)) {
+      return props.getProperty(propKeyRaw);
+    } else if (props.containsKey(propKeyFile)) {
+      try {
+        return new String(Files.readAllBytes(Paths.get(props.getProperty(propKeyFile))));
```

Comment:
I think spotbugs is complaining that this should be `new String(..., StandardCharsets.UTF_8)`